test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -343,14 +350,14 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 					return true
 				} else {
 					for _, podcpu := range podCpus.ToSlice() {
-						for _, cpu := range cpusNotinSchedulingDomains {
-							if !strings.Contains(cpu, fmt.Sprint(podcpu)) {
```

Comment:
This line checks if the `podcpu` is not substring of `cpu`.
podcpu is cpu id for example: `3` .
the `cpu` is single line output of the `/proc/schedstats` command, for example:
`[ cpu0 0 0 0 0 0 0 68186178574807 516377247436 331031573  cpu1 0 0 0 0 0 0 75970491002822 375072790117 330836684]`
The problem with this check is that if the string "3" not appears somewhere in this line, we return `true`, which is wrong, because although this line refer to `cpu0` and `cpu1` but the string `3` still appears.

Reply:
this deserves to be a comment in the code

Reply:
I don't think the test is going to pass. 
I think this patch just float that the issue is real and consistent.

Reply:
Right, there's probably little point in documenting this even in the fixing patch.